Apps/Sandcastle/gallery/development/Picking Sublayer Regions.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo - Picking Sublayer Regions</title>
+    <script type="text/javascript" src="../../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+    <style type="text/css">
+        @import url(../../templates/bucket.css);
```

Comment:
Moving this block by hand, by directly modifying a save file, has broken the styling in Sandcastle.  Need to put it back.